mojaloop/values.yaml

```diff
@@ -1811,7 +1841,9 @@ ml-api-adapter:
                       "queue.buffering.max.ms": 50,
                       "batch.num.messages": 100,
                       "api.version.request": true,
-                      "dr_cb": true
+                      "dr_cb": true,
+                      "socket.keepalive.enable": true,
+                      "socket.max.fails": 0
```

Comment:
I would suggest also adding the same kafka configs to the "ml-api-adapter" chart:

"socket.keepalive.enable": true
"socket.max.fails": 0